packages/map-template/src/hooks/useCurrentVenue.js

```diff
@@ -80,7 +80,18 @@ export const useCurrentVenue = () => {
     /**
      * Generate list of categories that exist on Locations in the current Venue.
      */
-    const updateCategories = () => {
+    const updateCategories = async () => {
+        const categories = await window?.mapsindoors?.services?.SolutionsService?.getCategories();
+        const mainmenu = appConfig?.menuInfo?.mainmenu ?? [];
```

Comment:
```suggestion
        const mainMenu = appConfig?.menuInfo?.mainmenu ?? [];
```

Let's not continue the misspelling 😁